docs/src/community/index.rst

```diff
@@ -46,3 +47,23 @@ smoother interoperability:
    :hidden:
 
    iris_xarray
+
+Plugins
+-------
```

Comment:
Please could this be a new `docs/src/community/plugins.rst`? I can picture the page expanding to list plugins that we know about, and provide instruction on how to create a plugin.

Would still be good to have the heading in this file (`community/index.rst`), but just with a single sentence and a link.

Reply:
Went ahead and included a section on creating a plugin straight away, because it seemed a bit bare without.  Granted there may be more to add once I've gone through the whole process, but I hope it's a good start.

Reply:
It's looking great, thanks 🙂